docs/volume/volume-snapshots.md

```diff
@@ -8,33 +8,48 @@ keywords:
+You can create a volume snapshot from an existing volume by following these steps:
 
-:::
+1. Go to the **Volumes** page.
+1. Choose the volume that you want to take a snapshot of and select **⋮ > Take Snapshot**.
+
+ ![create-volume-snapshot-1](/img/v1.2/volume/create-volume-snapshot-1.png)
+1. Enter a **Name** for the snapshot.
+
+ ![create-volume-snapshot-2](/img/v1.2/volume/create-volume-snapshot-2.png)
+1. Select  **Create** to finish creating a new volume snapshot.
 
-After creating a volume, you can create volume snapshots by following the steps below:
+1. You can view all VM snapshots and check the status of this operation from the **Backup & Snapshot > VM Snapshots** page. When the **ReadyToUse** status becomes **true**, the snapshot is created successfully.
 
-1. Click the `⋮` button and select the `Take Snapshot` option.
-1. Configure the `Name` of the new image and click `Create`.
+:::note
 
-![create-volume-snapshot-1](/img/v1.2/volume/create-volume-snapshot-1.png)
-![create-volume-snapshot-2](/img/v1.2/volume/create-volume-snapshot-2.png)
+A recurring snapshot is currently not supported and is tracked via [harvester/harvester#572](https://github.com/harvester/harvester/issues/572).
 
-## Restore a New Volume using Volume Snapshot
+:::
 
-After creating a volume snapshot, you can restore a new volume using the volume snapshot by following the steps below:
+## Restore a new volume from a volume snapshot
 
-1. Go to the `Backup & Snapshot > Volume Snapshots` page or the `Snapshots` tab in each `Volumes` Detail page.
-1. Click the `⋮` button and select the `Restore` option.
-1. Specify the `Name` of the new Volume.
-1. If the source volume is not an image volume, you can also select a different `StorageClass`.
-1. Click `Create`.
+You can restore a new volume from an existing volume snapshot by following these steps:
 
+1. Go to the **Backup & Snapshot > Volume Snapshots** page or go to the **Volumes** page and select the **Snapshots** tab.
+1. Select **⋮ > Restore**.
 ![restore-volume-snapshot-1](/img/v1.2/volume/restore-volume-snapshot-1.png)
 ![restore-volume-snapshot-2](/img/v1.2/volume/restore-volume-snapshot-2.png)
+1. Specify the **Name** of the new volume.
 ![restore-volume-snapshot-3](/img/v1.2/volume/restore-volume-snapshot-3.png)
+1. If the source volume is not an image volume, you can select a different **StorageClass**. You can not change the **StorageClass** if the source volume is an image volume.
 ![restore-volume-snapshot-4](/img/v1.2/volume/restore-volume-snapshot-4.png)
+1. Select **Create** to finish restoring a new volume.
+
+### View the replication and health status in Longhorn
+
+1. You can view the replication and health status for the new volume from the Longhorn dashboard. The Longhorn dashboard is available directly in the Harvester dashboard, but you must first enable developer tools & features. Go to the **Preference** page.
```

Comment:
the replication and health status for volumes are independent of volume snapshots. Prefer to add a new page.
And we should note that we only support using the embedded Longhorn dashboards for debugging and validation purposes.

Reply:
@futuretea Removing this section and opening a PR to add a page covering how to view replication and health status for volumes via the embedded longhorn dashboard